### PR TITLE
bump runtime to GNOME 40

### PR DESCRIPTION
**Pull request by muelli**
Let's see whether this builds and runs

### CODE REVIEW

**Comment by muelli** [comment]
```
flatpak-builder: Downloading ffmpegDownloading http://archive.ubuntu.com/ubuntu/pool/universe/f/ffmpeg/ffmpeg_4.1.1.orig.tar.xz
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
Failed to download sources: module ffmpeg: The requested URL returned error: 404 Not Found
```

**Comment by lionirdeadman** [comment]
Yes, all the modules are from Ubuntu archives that are dead. I'm working on it.

File: de.create3000.titania.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
 	"app-id": "de.create3000.titania",
 	"branch": "stable",
```
Comment (hfiguiere) [inline de.create3000.titania.json:3]:
```suggestion
```

**Comment by hfiguiere** [comment]
You also probably need to deal with https://github.com/flathub/flathub/issues/2180, not sure.

**Pull request closed without merging**